Lib/vanilla/__init__.py

```diff
@@ -70,9 +70,11 @@
 
 __version__ = "0.1"
 
-# OS 10.7 objects
+# NSGridview is available from OS 10.12+
 try:
-    from vanilla.vanillaPopover import Popover
-    __all__.append("Popover")
-except (ImportError, NameError):
+    from AppKit import NSGridView
+except AttributeError:
```

Comment:
How would this be an AttributeError? Shouldn't it be an ImportError?

Reply:
Beat me... try

```python
import AppKit

AppKit.NSDoesNotExists
```

Reply:
Yes that would be an AttributeError. But that's not what the code does.